### ISSUE TITLE
Unblock ajax.googleapis.com

### ISSUE THREAD

**Issue by HalloTschuess** [StevenBlack/hosts#2516]
Seems like ajax.googleapis.com slipped in again. A website I use doesn't work anymore, because it uses jQuery loaded from this domain. I'm sure there are many others as well.

It was already removed once in 0b32cf2fd87899bc08b5bf5bf0c6be9ef42f3f88

**Comment by XhmikosR** [comment]
@StevenBlack this one too seems like a mistake

**Comment by domdfcoding** [comment]
For me it is affecting some (but strangely not all) stackexchange sites.

**Comment by russeg** [comment]
why block this?

**Comment by StevenBlack** [comment]
This is resolved in commit 00f37ec9.

**Comment by textures2** [comment]
> This is resolved in commit [00f37ec](https://github.com/StevenBlack/hosts/commit/00f37ec9e87ec65b1bfdb8ad46163efb9af5fa69).

Thanks but that commit is from 2022.  What's puzzling is that this showed up in pihole sometime over the past 48h, trying to piece together how it happened:
```
pi@pihole ➜  ~ pihole -q -exact ajax.googleapis.com
 Exact match for ajax.googleapis.com found in:
  - https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts
pi@pihole ➜  ~ pihole -w ajax.googleapis.com
  [i] Adding ajax.googleapis.com to the whitelist...
  [✓] Reloading DNS lists
```
Of course I can clearly see when I go look at `https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts` that there is not an entry there presently.

**Comment by StevenBlack** [comment]
Peter @textures2 that commit is from 2 days ago.

**Comment by textures2** [comment]
> Of course I can clearly see when I go look at `https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts` that there is not an entry there presently.

I found the log file on my raspberry pi corresponding to when it updates its blacklist/denylist, a couple days ago:
```
-rw-r--r-- 1 root   root       1229 Nov 26 03:49 pihole_updateGravity.log
```
The log indicates it retrieved and synced the file:

```
  [i] Target: https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts
  [✓] Status: Retrieval successful
  [✓] Parsed 160097 exact domains and 0 ABP-style domains (ignored 1 non-domain entries)
      Sample of non-domain entries:
        - "0.0.0.0"

  [i] List has been updated
```
That update suggests the list had 160097 entries.

I just went and pulled it down right now and see a LOT more:

```
pi@pihole ➜  ~ wget https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts
--2023-11-28 23:36:16--  https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.111.133, 185.199.108.133, 185.199.110.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.111.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 4914335 (4.7M) [text/plain]
Saving to: ‘hosts’
2023-11-28 23:36:17 (3.90 MB/s) - ‘hosts’ saved [4914335/4914335]

pi@pihole ➜  ~ wc -l hosts
166954 hosts
```

TL;DR - raw.githubusercontent.com/StevenBlack/hosts/master/hosts looks like it was serving stale data??  How is that possible.

**Comment by textures2** [comment]
To be sure: I may be misunderstanding and a simple linecount is not an accurate representation of # of entries?

**Comment by textures2** [comment]
Okay looks like my misunderstanding.

> Peter @textures2 that commit is from 2 days ago.

I see that now, my apologies.  This explains it conclusively I suppose.  Sorry for the churn, late where I am.
